stack.yaml

```diff
@@ -10,6 +10,10 @@ packages:
 - hie-ghc-tree
 - hie-hare
 - hie-docs-generator
+- location:
```

Comment:
Why is this necessary? I don’t see any obvious changes to the HaRe-plugin that would require this.